update readme to include parquet files and clarification about samples (#29)
CDCgov/covid19-forecast-hub: model-output/README.md

### Metadata

> The metadata file will be saved within the model-metdata directory in the Hub's GitHub repository, and should have either of the following naming convention:

Reviewer: ```suggestion
The metadata file will be saved within the model-metdata directory in the Hub's GitHub repository. It should be a YAML file with base name `{team name}-{model name}`, and extension `.yml` or `.yaml`, e.g.
```

>     team-model.yml

Reviewer: ```suggestion
    exampleteam-examplemodel.yml
    otherteam-othermodel.yaml
```

>     team-model.yaml

Reviewer: ```suggestion
```

Each forecast file should have the following
format

>     {YYYY-MM-DD}-{team}-{model}.csv
> 
>     {YYYY-MM-DD}-{team}-{model}.parquet

Reviewer: ```suggestion
    {YYYY-MM-DD}-{team}-{model}.csv

or

    {YYYY-MM-DD}-{team}-{model}.parquet

depending on whether the team is submitting forecasts as `.csv` files or as `.parquet` files.
```

>     YYYY-MM-DD-team-model.csv

Reviewer: ```suggestion
    {YYYY}-MM-DD}-{team name}-{model name}.csv
```

>     YYYY-MM-DD-team-model.parquet

Reviewer: ```suggestion
    {YYYY-MM-DD date}-{team name}-{model name}.parquet
```

Reviewer: and then add a concrete example, e.g. 

`2024-11-16-exampleteam-examplemodel.parquet`

Author: On second thought, I think this is better? 

```suggestion
   {YYYY-MM-DD}-{team}-{model}.parquet
```
since we specify above that `team` is the team name and
 `model` refers to the name of the model.
+ concrete example

> -   "quantile" or
> -   "samples".

Reviewer: And change "are either" above "should be one of"

```suggestion
-   `quantile` 
-   `samples`
```
